Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -361,6 +363,7 @@ local questsToBlacklistBySoDPhase = {
         [90200] = true, -- Hiding Rogue Mutilate Dun Morogh for now as there are too many icons
         [90201] = true, -- Hiding Rogue Shiv Duskwood for now as there are too many icons
         [90204] = true, -- Hiding Shaman Way of Earth Silverpine Forest for now as there are too many icons
+        [90217] = true, -- Hiding Shaman Molten Blast Mulgore for now as there are too many icons
```

Comment:
```suggestion
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5611/commits/41d028fb6a8d634631939db7bb3efe53f6abc5ef